clients/corral/corral.go

```diff
@@ -68,9 +68,9 @@ func SetCustomRepo(repo string) error {
 	return nil
 }
 
-// CreateCorral creates a corral taking the corral name, the package path, and a debug set so if someone wants to view the
+// CreateCorralAndWait creates a corral taking the corral name, the package path, and a debug set so if someone wants to view the
 // corral create log
-func CreateCorral(ts *session.Session, corralName, packageName string, debug bool, cleanup bool) ([]byte, error) {
+func CreateCorralAndWait(ts *session.Session, corralName, packageName string, debug bool, cleanup bool) ([]byte, error) {
```

Comment:
nit this can be shortened to`func CreateCorralAndWait(ts *session.Session, corralName, packageName string, debug, cleanup bool)`